docs/project/find-an-issue.md

```diff
@@ -107,9 +103,9 @@ In this section, you find and claim an open documentation lines issue.
 
     ![Open issues](/project/images/issue_list.png)
 
-3. Look for the <strong class="gh-label beginner">exp/beginner</strong> items on the list.
+3. Look for the <strong class="gh-label beginner">exp/beginner</strong> issues.
```

Comment:
@tombee Good point. Maybe the thing to do is #4 and then have them look in the sorted list for "kind/writing" or some other kind.  @lyndaoleary your choice.